src/context/PlanetProvider.tsx

```diff
@@ -18,10 +18,24 @@ const PlanetProvider = ({
   const [state, dispatch] = useReducer(PlanetReducer, initialState);
 
   useEffect(() => {
-    // TODO: crear servicio fetch para obtener planeta
-    dispatch({ type: PlanetActions.LOAD_PLANET });
-    const item = data.find((e) => e.name.toLowerCase() === planetName);
-    dispatch({ type: PlanetActions.LOAD_PLANET_SUCCESS, payload: item });
+    void (async () => {
+      dispatch({ type: PlanetActions.LOAD_PLANET });
+
+      const res = await ServicePlanets.get(planetName);
+      const { planet, error } = res;
+
+      if (planet) {
+        dispatch({
+          type: PlanetActions.LOAD_PLANET_SUCCESS,
+          payload: planet,
+        });
+      }
+
+      dispatch({
+        type: PlanetActions.LOAD_PLANET_ERROR,
+        payload: error,
+      });
```

Comment:
No debería ser algo tipo? (solo dispatch con payload Error is hay error?)
```suggestion
      const { planet, error } = res;

      if (error) {
	      dispatch({
	        type: PlanetActions.LOAD_PLANET_ERROR,
	        payload: error,
	      });
      }
    
      if (planet) {
        dispatch({
          type: PlanetActions.LOAD_PLANET_SUCCESS,
          payload: planet,
        });
      }
```

Reply:
Desconozco el uso de dispatch, asi que improvise el error basandome en el success que existia